### ISSUE TITLE
Tasks Executer

### ISSUE THREAD

**Issue by hadialaddin** [hadialaddin/bybit-risk-monitor#8]
Uses a First-In-First-Out (FIFO) approach to execute the Tasks:
- Execute task
- Add to Notifications Queue with message "Task ID executed on timestamp"

Tasks to support:
- Market Open "%" of available_balance.
- Market Open "QUANTITY of "SHORT/LONG" position
- Market Close "QUANTITY/%" of "SHORT/LONG" position

NOTE: If Hedge Mode is not enabled, a "CLOSE" is an order with "Reduce-Only".

**Comment by JerryVdV** [comment]
Maybe set the condition of "Market Open "%" of available_balance" with an function that is based on % of portfolio (and if that amount is not available to then use the available balance). 

That should help with ensuring that profitable trades make up for lost trades in the win/lose ratio.

**Comment by hadialaddin** [comment]
@JerryVdV You mean to divide balance into portfolio as well? (Eg. always take 30% of total Balance as Solana positions with a max of $ value at any time)?

**Comment by JerryVdV** [comment]
Yes sir, otherwise the positions will get smaller the more there are open
at any given time. In theory making it harder to recuperate losing trades
(if those had a larger position in terms of the margin).

Just a thought off course.

**Comment by hadialaddin** [comment]
Got it, nice idea. Will add as a separate ticket.

**Comment by amesas** [comment]
Are you planning to try limit close?

**Comment by JerryVdV** [comment]
Would be nice yes.

**Comment by hadialaddin** [comment]
@amesas Yes on the list
